.dockerignore

```diff
@@ -0,0 +1,3 @@
+project-api/node_modules
+project-ui/build
+project-ui/node_modules
```

Comment:
Without a `.dockerignore`, you'd end up copying things that should stay local onto your device. This makes the initial file copy to your device take much longer than necessary. Also, `node_modules` contents depends on the OS so you should execute `npm install` within the container and not assume that what is installed on your local system is a match. `build` etc should also be ignored as builds should be done with deps installed in the container.